returns/transducers/tfilter.py

```diff
@@ -0,0 +1,41 @@
+from typing import Callable, TypeVar
+
+_ValueType = TypeVar('_ValueType')
+_AccValueType = TypeVar('_AccValueType')
+
+
+def tfilter(
+    predicate: Callable[[_ValueType], bool],
+) -> Callable[
+    [Callable[[_AccValueType, _ValueType], _AccValueType]],
+    Callable[[_AccValueType, _ValueType], _AccValueType],
+]:
+    """
+    :py:func:`filter <filter>` implementation on a transducer form.
```

Comment:
Let's make the first line more user-friendly by removing `:py:func:` stuff.
It can be later in the body.